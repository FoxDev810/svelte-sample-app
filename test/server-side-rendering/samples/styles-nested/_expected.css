div[svelte-4188175681], [svelte-4188175681] div {
	color: red;
}

div[svelte-146600313], [svelte-146600313] div {
	color: green;
}

div[svelte-1506185237], [svelte-1506185237] div {
	color: blue;
}
