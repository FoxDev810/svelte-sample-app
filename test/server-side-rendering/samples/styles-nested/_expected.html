<div svelte-1408461649>red</div>
<div svelte-54999591>green: foo</div>
<div svelte-2385185803>blue: foo</div>
<div svelte-54999591>green: bar</div>
<div svelte-2385185803>blue: bar</div>